packages/beacon-node/test/unit/util/graffiti.test.ts

```diff
@@ -23,7 +23,7 @@ describe("Graffiti helper", () => {
     ];
     for (const {input, result} of cases) {
       it(`Convert graffiti UTF8 ${input} to Buffer`, () => {
-        expect(toGraffitiBuffer(input).toString("hex")).toBe(result);
+        expect(Buffer.from(toGraffitiBytes(input)).toString("hex")).toBe(result);
```

Comment:
why do we need `Buffer.from` here now? we just changed the function name and type but shouldn't matter?

Reply:
Ah right, misread the code, makes sense